app/panel/components/Subscribe.jsx

```diff
@@ -38,7 +38,7 @@ const Subscribe = ({ match }) => {
 			<div className="pitch-container">
 				<span className="pitch-text" dangerouslySetInnerHTML={{ __html: t('subscribe_pitch') }} />
 			</div>
-			<PanelToTabLink href="https://www.ghostery.com/products/plus?utm_source=gbe&utm_campaign=in_app_plus_lm">
+			<PanelToTabLink href="https://www.ghostery.com/pricing/">
```

Comment:
Let's restore the utm params here

Reply:
Also, let's update the url to use `GHOSTERY_BASE_URL` to link to stage/prod as appropriate (as per Tino)

Reply:
Restored and updated link!